### DISCUSSION TITLE
Conda install does not install the latest version

### DISCUSSION THREAD

**Discussion by danieljacobsen-ai** [meta-pytorch/botorch#1206] [Q&A] [answered]
conda install botorch -c pytorch -c gpytorch installs botorch 0.6.2 on my win10.

conda install botorch=0.6.4 -c pytorch -c gpytorch (or 0.6.3) gives

"The following packages are not available from current channels:" 

Is there some channel I need to add to allow Conda to actually install the latest version of Botorch?

Thanks.

**Comment by danieljacobsen-ai** [comment, marked as answer]
conda install -c conda-forge botorch solves the problem. 

Perhaps the installation instructions should be updated accordingly?

**Reply by saitcakmak** [reply]
Thanks for flagging this! #1207 updates the instructions.

**Comment by Balandat** [comment]
Hmm this is odd. Using the pytorch channel is the recommended way of installing botorch from conda (this is maintained by us, the conda-forge version is a copy). The conda package on the pytorch channel is up to date and a noarch package and so the fact that this is windows shouldn't really matter. 

I see you mixed the order of the flags and the positional args - I don't think this *should* matter, but what happens if you do 
`conda install -c pytorch -c gpytorch botorch`?

cc @saitcakmak who made the latest release.

**Reply by saitcakmak** [reply]
This is strange. It is not working for me either, i.e., I cannot conda install botorch from the pytorch channel regardless of the version.

**Reply by saitcakmak** [reply]
Looks like that was because I removed the `gpytorch` channel. With that in place, I can install 0.6.2 but nothing newer. 

I think this might be because we need `pyro-ppl` since 0.6.3, which is only available on the `conda-forge` channel.
Doing `conda install botorch==0.6.4 -c pytorch -c conda-forge` works and installs botorch from pytorch channel and the rest from conda-forge.
